frontend/assets/lang/pl.json

```diff
@@ -15,5 +15,48 @@
         "ComboboxField": "Wybierz kierunek",
         "ComboboxDepartment": "Wybierz wydział",
         "ComboboxSpecialization": "Wybierz specjalizację"
+    },
+    "SignIn":{
+        "Header": "Zaloguj się na swoje konto",
```

Comment:
I'm not convinced to this.

Reply:
Ive changed it to just "Logowanie" and "Sign in"